.github/workflows/wasmcloud-chart.yml

```diff
@@ -0,0 +1,83 @@
+name: wasmcloud-chart
+
+env:
+  HELM_VERSION: v3.13.1
+
+on:
+  push:
+    tags:
+      - 'chart-v[0-9].[0-9]+.[0-9]+'
+  pull_request:
+    paths:
+      - 'chart/**'
+      - '.github/workflows/wasmcloud-chart.yml'
+
+jobs:
+  validate:
+    runs-on: ubuntu-22.04
+
+    steps:
+      - uses: actions/checkout@v4.1.1
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v3.5
+        with:
+          version: ${{ env.HELM_VERSION }}
+
+      # Used by helm chart-testing below
+      - name: Set up Python
+        uses: actions/setup-python@v4.7.1
+        with:
+          python-version: '3.12.0'
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.4.0
+        with:
+          version: v3.9.0
+          yamllint_version: 1.32.0
+          yamale_version: 4.0.4
+
+      - name: Run chart-testing (lint)
+        run: |
+          ct lint --charts chart/ --validate-maintainers=false
+
+      - name: Create kind cluster
+        uses: helm/kind-action@v1.8.0
+
+      - name: Run chart-testing (install)
+        run: ct install --charts chart/
+
+  publish:
+    if: ${{ github.event_name == 'push' && startsWith(github.ref, 'refs/tags/chart-v') }}
```

Comment:
Nit: isn't checking the event name redundant? I'm not sure there's a way for the ref to be a tag without the event being a push

Reply:
> Nit: isn't checking the event name redundant? I'm not sure there's a way for the ref to be a tag without the event being a push

It is, I'll drop the `github.event_name` check here.